examples/utils.py

```diff
@@ -62,6 +62,18 @@ def load_dataset(args):
   elif args['featurizer'] == 'GC':
     from deepchem.feat import MolGraphConvFeaturizer
     featurizer = MolGraphConvFeaturizer()
+  elif args['featurizer'] == 'AC':
+    from deepchem.feat import AtomicConvFeaturizer
+    featurizer = AtomicConvFeaturizer(
+        frag1_num_atoms=100,
+        frag2_num_atoms=1000,
+        complex_num_atoms=1100,
+        max_num_neighbors=12,
+        neighbor_cutoff=4)
```

Comment:
I wonder if we should treat `max_num_neighbors` and `neighbor_cutoff` as hyperparameters as well.

Reply:
I think that would be best, but it would involve re-featurizing pdbbind for every iteration of model training, which would be pretty time-consuming. These values are taken from the original ACNN paper.